deliverable-2/BUG-ANALYSIS.md

```diff
@@ -96,17 +97,19 @@ NOTE: We may have messed up this bug because we forked into our own repo by acci
 
 The reason for [Issue #1220] “unmute this page”:
     
-   Unmute this page doesn't work” is one of the least complicated issues among our selection. The team was able to identify the change really quickly solely based on previous knowledge. It is also the issue that we didn’t identify any foreseeable risks because its scope is narrowed to a single change in a single module. So it is our first pick for its safety. Also the anticipated work is less than an hour. Lastly the team agrees that this change is very likely to be accepted. 
+   *“‘Unmute this page’ doesn't work” is one of the least complicated issues among our selection. The team was able to come up with a rough idea on the general direction of the change relatively quickly based on our knowledge. It is also an issue that we didn’t identify any foreseeable risks because its scope is narrowed to a change in a single module. Therefore, it became our first pick as we believe it is a beginner friendly issue. With the combination of reproducing the bug, identifying the issue, locating the appropriate file to fix and writing test cases, we anticipated work is around an hour or two. In addition, this bug is critical for the project. Failing to unmute the soundtrack will cause failure in other customized services such as Youtube, Spotify and New York Times in which their main interaction with users are through audio or videos. Therefore, even though it is not a difficult fix in a technical aspect, it is an important feature for the project.*
```

Comment:
Some suggestions:

Unmute this page’ doesn't work” is one of the ~~least complicated~~ more reasonable issues among our selection in terms of difficulty and complexity.